packages/central-server/src/apiV2/import/importEntities/extractEntitiesByCountryName.js

```diff
@@ -52,13 +69,27 @@ const processXlsxRow = (row, { countryName }) => {
 const xlsxParser = filePath => {
   const workbook = xlsx.readFile(filePath);
   return Object.entries(workbook.Sheets).reduce(
-    (entitiesByCountry, [countryName, sheet]) => ({
-      ...entitiesByCountry,
-      [countryName]: xlsx.utils
-        .sheet_to_json(sheet)
-        .map(row => processXlsxRow(row, { countryName })),
-    }),
-    {},
+    ({ entitiesByCountry, entitiesWithoutCountry }, [countryName, sheet]) => {
```

Comment:
Is `entitiesByCountry` originally contains `entitiesByCountry` and `entitiesWithoutCountry`? Or it has something to do with the changes for `geojsonParser`?